### DISCUSSION TITLE
Kubero fetcher and self-signed TLS certificates

### DISCUSSION THREAD

**Discussion by fqxp** [kubero-dev/kubero#346] [Q&A]
I am trying to use the latest version of Kubero with a self-hosted git repository (gitea). The gitea server is reachable through a TLS (https)  connection. Since the certificate is signed with my personal CA, I need to somehow tell the Kubero fetcher about the root certificate. Unfortunately, I can‘t find any option to do this. 

Is there a feature for this in Kubero that I have missed, or is there another way to do this? May anyone share their experiences with this problem?

Also, there I may get the same problem with the registry, which is also reachable via https, but I haben’t gotten that far yet.

**Comment by mms-gianni** [comment]
You'd definitely run into issues with the registry. But that's caused by kubernetes since they are usually configured not to pull from "insecure" registries. As far as I know, there is no other way around, other than changing the docker/containerd configuration 
https://medium.com/@nash.checkin/kind-kubernetes-in-docker-pulling-image-from-private-insecure-registry-2711706629cb 

But for Kubero it might be configurable. I think this is worth opening an issue to add this option. Should not be a big deal. 

Sorry for the late reply. 😄
